data_structures/linked_list/__init__.py

```diff
@@ -2,21 +2,24 @@ class Node:
     def __init__(self, item, next):
         self.item = item
         self.next = next
-
 class LinkedList:
     def __init__(self):
         self.head = None
+        self.size=0
```

Comment:
```suggestion
        self.size = 0
```
psf/black is just going to undo all this text with no room to breathe.